gnd/build.gradle

```diff
@@ -103,6 +107,49 @@ android {
     hilt {
         enableTransformForLocalTests = true
     }
+
+    // ErrorProne and NullAway configuration.
```

Comment:
Can you please add a link to the wiki/github repo of this plugin?

Reply:
Done